go-controller/pkg/config/config.go

```diff
@@ -57,16 +57,17 @@ var (
 
 	// Default holds parsed config file parameters and command-line overrides
 	Default = DefaultConfig{
-		MTU:               1400,
-		ConntrackZone:     64000,
-		EncapType:         "geneve",
-		EncapIP:           "",
-		EncapPort:         DefaultEncapPort,
-		InactivityProbe:   100000, // in Milliseconds
-		OpenFlowProbe:     180,    // in Seconds
-		MonitorAll:        true,
-		LFlowCacheEnable:  true,
-		RawClusterSubnets: "10.128.0.0/14/23",
+		MTU:                   1400,
+		ConntrackZone:         64000,
+		EncapType:             "geneve",
+		EncapIP:               "",
+		EncapPort:             DefaultEncapPort,
+		InactivityProbe:       100000, // in Milliseconds
+		OpenFlowProbe:         180,    // in Seconds
+		OfctrlWaitBeforeClear: 0,      // in Milliseconds
```

Comment:
before this option was added in ovn, was the controller clearing immediately?

Reply:
yes,  and still the default behavior.